activerecord/lib/active_record/connection_adapters/abstract/schema_definitions.rb

```diff
@@ -169,6 +169,7 @@ def primary_key(name, type = :primary_key, **options)
         :date,
         :datetime,
         :decimal,
+        :numeric,
```

Comment:
Still not quite right. These don't need to be separate types, just aliases. I think this should just be `alias_method :numeric, :decimal`, which lets us remove the additional `if` statement below.